tox/constants.py

```diff
@@ -0,0 +1,56 @@
+"""All non private names (no leading underscore) here are part of the tox API.
+
+They live in the tox namespace and can be accessed as tox.[NAMESPACE.]NAME
+"""
+import sys as _sys
+
+
+class PYTHON:
+    CPYTHON_VERSION_TUPLES = [(2, 7), (3, 4), (3, 5), (3, 6), (3, 7)]
+    OTHER_PYTHON_INTERPRETERS = ['jython', 'pypy', 'pypy3']
+    _map = {'py': _sys.executable, 'py2': 'python2', 'py3': 'python3'}
+    _map.update({'py%s%s' % (major, minor): 'python%s.%s' % (major, minor)
+                 for major, minor in CPYTHON_VERSION_TUPLES})
+    _map.update({interpreter: interpreter for interpreter in OTHER_PYTHON_INTERPRETERS})
+    DEFAULT_FACTORS = _map
+    CURRENT_RELEASE_ENV = 'py36'
+    """Should hold currently released py -> for easy updating"""
+    QUICKSTART_PY_ENVS = ['py27', 'py34', 'py35', CURRENT_RELEASE_ENV, 'pypy', 'jython']
+    """For choices in tox-quickstart"""
```

Comment:
variable doc comments come before not after the variable

Reply:
Using normal comments then. A bit sad that this is not supported properly, as I like that style more than comments.

Reply:
I know, but we want to be able to render the description from here in a autodoc entries on our website, given this wants to be public.